src/Raven.Server/ServerWide/Commands/ClusterTransactionCommand.cs

```diff
@@ -336,7 +336,11 @@ private void EnsureAtomicDocumentWrites(DatabaseTopology dbTopology, ClusterOper
                     case nameof(CommandType.PUT):
                         clusterTransactionDataCommand.Type = CommandType.CompareExchangePUT;
 
-                        var dynamicJsonValue = new DynamicJsonValue { ["Id"] = docId };
+                        var dynamicJsonValue = new DynamicJsonValue
+                        {
+                            [Constants.CompareExchange.ObjectFieldName] = new DynamicJsonValue { ["Id"] = docId }
```

Comment:
Does it require schema update?

Reply:
I don't believe so, since we never look inside the atomic guard, all we care about is the raft index, which isn't part of the value.